client/client.go

```diff
@@ -57,10 +58,9 @@ type Envelope struct {
 	Data json.RawMessage `json:"data"`
 }
 
-type Body struct {
```

Comment:
Am I understanding correctly that `Body` wasn't being used?

Reply:
Yup, that's right :)